src/Temporalio/Worker/TemporalWorkerOptions.cs

```diff
@@ -71,6 +71,13 @@ public TemporalWorkerOptions()
         /// </summary>
         public string? BuildID { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether this worker opts into the worker versioning feature. This ensures it only receives
+        /// workflow tasks for workflows which it claims to be compatible with. The <see cref="BuildID"/> field
+        /// is used as this worker's version when enabled, and must be set.
+        /// </summary>
+        public bool? UseWorkerVersioning { get; set; }
```

Comment:
```suggestion
        public bool UseWorkerVersioning { get; set; }
```

Unless there's value in `null` separate from `false`